src/helpers.js

```diff
@@ -323,7 +324,32 @@ const helpers = {
    */
   cleanupString(string) {
     return string.replace(/\s\s+/g, ' ').trim().toLowerCase();
-  }
+  },
+
+  /**
+   * Handle error for method transaction.prepareData
+   * Check if error is one of the expected and return the message
+   * Otherwise, throws the unexpected error
+   *
+   * @param {Error} e Error thrown
+   *
+   * @return {string} Error message
+   * @memberof Helpers
+   * @inner
+   */
+  handlePrepareDataError(e) {
```

Comment:
This is currently called in a lot of places where it might not be needed. It probably needs to be called in 1 place and we could keep the code there so it's easier to understand.

Reply:
This code was used before on the wallet desktop, I just moved here, in order to prevent duplicating code.

The method I should change to handle the errors is `transaction.prepareData`, however this is called in many places and this change would break compatibility, so I decided to keep the method unchanged and handle the errors like this. I don't see any problems with it, since it's being reused and without code duplication.

Reply:
Ok, now `const message = helpers.handlePrepareDataError(e);` is being used in the other places, so it makes sense to keep this code here.